server/controllers/finance/vouchers.js

```diff
@@ -266,15 +276,40 @@ function safelyDeleteVoucher(guid) {
     DELETE FROM document_map WHERE uuid = ?;
   `;
 
+  // NOTE(@jniles) - this is a naive way of undoing reversals.  If no value is
```

Comment:
Next time it will be better to use appropriate comment instruction for multi-line. it is fancy.